tests/data_context/test_get_data_context.py

```diff
@@ -114,10 +114,10 @@ def test_config_returns_base_context_overrides_yml(tmp_path: pathlib.Path):
     assert isinstance(context, BaseDataContext)
     assert context.expectations_store_name == "new_expectations_store"
 
-
+@pytest.mark.parametrize("ge_cloud_mode",[True, None])
```

Comment:
Nice parametrization!

Reply:
Could you also add separate explicit tests for ge_cloud_mode=False (with and without cloud env configured)?

Reply:
added in fff5241d9d2e5e130b870ad41b07d6b11e1868a9